Sources/UI/Chat/Composer/ComposerView.swift

```diff
@@ -275,7 +273,8 @@ public extension ComposerView {
     }
     
     internal func updateSendButton() {
-        let isAnyFileUploaded = uploadManager?.items.first(where: { $0.attachment != nil }) != nil
+        let isAnyFileUploaded = uploadManager?.images.first(where: { $0.attachment != nil }) != nil
+            || uploadManager?.files.first(where: { $0.attachment != nil }) != nil
```

Comment:
😱 this expression is massive. Mind splitting it into something more readable?

Reply:
Done. I really tried :)